apps/condo/domains/news/tasks/publishSharedNewsItemsByNewsItem.js

```diff
@@ -0,0 +1,20 @@
+const { find, getById } = require('@open-condo/keystone/schema')
+const { createTask } = require('@open-condo/keystone/tasks')
+
+const { publishSharedNewsItem } = require('@condo/domains/news/tasks')
+
+/**
+ * @param {string} newsItemId
+ * @param {Array} sharedNewsItemsIds
+ * @returns {Promise<void>}
+ */
+async function publishSharedNewsItemsByNewsItem (newsItemId, sharedNewsItemsIds) {
+    const newsItem = await getById('', { id: newsItemId })
```

Comment:
What is the expected behavior in this case?
```suggestion
    const newsItem = await getById('NewsItem', { id: newsItemId })
```

Reply:
haha will fix :)